### PR TITLE
Fixed output default in unify command

### PR DESCRIPTION
**Pull request by Shellyber**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/24057

## Description
Fixed the default output to be the package path.

## Screenshots
Before the fix: 
image

After the fix: 
image

## Must have
- [ ] Tests

### CODE REVIEW

**Comment by Itay4** [comment]
please add test

**Comment by Shellyber** [comment]
@Itay4 Added tests. Please review

File: demisto_sdk/commands/unify/tests/unifier_test.py [hunk 1]
```diff
@@ -455,3 +455,32 @@ def test_unify_script__docker45(self):
                                    'script-SampleScriptPackageSanityDocker45_45.yml')
 
         assert expected_yml_45 == actual_yml_45
+
+
+TESTS_DIR = f'{git_path()}/demisto_sdk/tests'
```
Comment (Itay4) [inline demisto_sdk/commands/unify/tests/unifier_test.py:460]: move this to the top of the code

File: demisto_sdk/commands/unify/tests/unifier_test.py [hunk 2]
```diff
@@ -455,3 +455,32 @@ def test_unify_script__docker45(self):
...
+
+
+def test_unify_default_output_integration():
+    """Test unify command with default output - integration"""
```
Comment (Itay4) [inline demisto_sdk/commands/unify/tests/unifier_test.py:464]: update according to `Given-When-Then` template

File: demisto_sdk/commands/unify/tests/unifier_test.py [hunk 3]
```diff
@@ -455,3 +455,32 @@ def test_unify_script__docker45(self):
...
+    yml_files = unifier.merge_script_package_to_yml()
+    export_yml_path = yml_files[0]
+    expected_yml_path = TESTS_DIR + '/test_files/Packs/DummyPack/Integrations/UploadTest/integration-UploadTest.yml'
+
+    assert export_yml_path == expected_yml_path
+    os.remove(expected_yml_path)
+
+
+def test_unify_default_output_script():
+    """Test unify command with default output - script"""
```
Comment (Itay4) [inline demisto_sdk/commands/unify/tests/unifier_test.py:477]: update according to `Given-When-Then` template

File: demisto_sdk/commands/unify/tests/unifier_test.py [hunk 4]
```diff
@@ -455,3 +455,32 @@ def test_unify_script__docker45(self):
...
+    unifier = Unifier(input_path_integration)
+    yml_files = unifier.merge_script_package_to_yml()
+    export_yml_path = yml_files[0]
+    expected_yml_path = TESTS_DIR + '/test_files/Packs/DummyPack/Integrations/UploadTest/integration-UploadTest.yml'
+
+    assert export_yml_path == expected_yml_path
+    os.remove(expected_yml_path)
+
+
+def test_unify_default_output_script():
```
Comment (Itay4) [inline demisto_sdk/commands/unify/tests/unifier_test.py:476]: this test should be in `TestMergeScriptPackageToYMLScript` class

File: demisto_sdk/commands/unify/tests/unifier_test.py [hunk 5]
```diff
@@ -455,3 +455,32 @@ def test_unify_script__docker45(self):
                                    'script-SampleScriptPackageSanityDocker45_45.yml')
 
         assert expected_yml_45 == actual_yml_45
+
+
+TESTS_DIR = f'{git_path()}/demisto_sdk/tests'
+
+
+def test_unify_default_output_integration():
```
Comment (Itay4) [inline demisto_sdk/commands/unify/tests/unifier_test.py:463]: this test should be in `TestMergeScriptPackageToYMLIntegration` class

**Comment by Itay4** [comment]
also, please update changelog

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,4 +1,5 @@
 # Changelog
+* Fixed a bug when an output argument was not passed to the **unify** command.
```
Comment (Itay4) [inline CHANGELOG.md:2]:
```suggestion
* Fixed a bug in the **unify** command when output path was provided empty.
```

File: demisto_sdk/commands/unify/tests/unifier_test.py [hunk 7]
```diff
@@ -453,5 +476,26 @@ def test_unify_script__docker45(self):
 
         expected_yml_45 = get_yaml('demisto_sdk/tests/test_files/Unifier/SampleScriptPackage/'
                                    'script-SampleScriptPackageSanityDocker45_45.yml')
-
         assert expected_yml_45 == actual_yml_45
+
+    def test_unify_default_output_script(self):
+        """
+        Given
+        - DummyPack script valid Pack.
```
Comment (Itay4) [inline demisto_sdk/commands/unify/tests/unifier_test.py:484]:
```suggestion
        - DummyScript script.
        - No output path.
```

File: demisto_sdk/commands/unify/tests/unifier_test.py [hunk 8]
```diff
@@ -381,6 +383,27 @@ def test_unify_integration__detailed_description_with_yml_structure(self):
         assert expected_yml == actual_yml
         assert actual_yml['detaileddescription'] == description
 
+    def test_unify_default_output_integration(self):
+        """
+        Given
+        - DummyPack integration valid Pack.
```
Comment (Itay4) [inline demisto_sdk/commands/unify/tests/unifier_test.py:389]:
```suggestion
        - UploadTest integration.
        - No output path.
```

**Pull request merged**